### DISCUSSION TITLE
Learning and earning by doing app

### DISCUSSION THREAD

**Discussion by Goosie** [pandoracore/bitcoin-pro#11] [Ideas]
What do you think of the idea to create a sort of learning by doing app. An app where a user will learn step by step what kind of things you can do and build with Bitcoin-Pro together with MyCitadel wallet. In this process, a user can for example earn different tokens, certificates, etc.. Users will learn how you can program tokens etc. I know it's early and not everything is ready, but would like to work on it as soon it's possible. Can we create a team for this? We could make a business model from this. For example a user can buy a test and test if everything is understood, every wrong question goes to the makers of this project, of course we make a DAO in RGB to share the profits :-)

**Comment by UkolovaOlga** [comment]
Thank you for your interest in MyCitadel, Bitcoin Pro and RGB! :)

Your idea is great, though I should add some comments to it:

1. Overall, people, unless working in software development professionally, are not able to read what requires to be tested and are not able to provide proper and thorough feedback (often - even a simple bug report). This usually creates a lot of noise and spam compared to the actual valuable feedback. For handling this type of info flood, a team of people will be required from the side of software creators. Thus far, as a private company, Pandora Core was not planning to expand the team to cover this situation.

2. Issuing a lot of tokens and certificates in big amounts now (including the ones that will be issued by testers during their testing) would create an perception of the whole RGB effort being about tokens and ICO-like scams instead of describing what it actually is, creating a lot of fraudulent ICO-like aura around the project. As a private company, and as the core development team behind the whole RGB effort and LNP/BP Standards Association, we are not interested in distorting the idea and principles we have been working hard on. If someone else wants to go for it - they can (and will), but neither Bitcoin Pro, nor MyCitadel as commercial products can be used for that and our team will in no way be liable for any consequences.

3. As far as I understood from your idea, certificates/rewards for participation etc will need to be eventually exchanged/converted to something, which creates a problem of secondary market required, without even saying about possible Ponzi schemes :D 
Also, right now I can't imagine why one would need any tokens if testing can be done as common bug bounty program, for sats. If what you are talking about is an idea of reputation system (especially if they want to use LNP/BP Association's DID proposal) - that's a different story, but still, as it was with ICOs - one should think twice if they need a token for their platform to operate.
 In any case, if someone wants to do it - they surely can (and probably will). But at this moment I would not use our products as a starting point - those who want to build something will need to do it on their own and are free to do so.

4. In RGB you don't 'program tokens'. You either use a Schema (or create one yourself), or you use tools like Bitcoin Pro and others that can/will appear on the market, for assets issuance. So, learning how to program assets is a misleading task/goal.

5. As commercial company, Pandora Core has its own goals, roadmaps and targets. We can't allocate our own resources on building something that does not align with our strategy. Though, if after rounds of discussions, some shared interests would appear, we can join efforts with other players, of course.

6. DAO is possible even with Bitcoin :) Sharing profits does not require RGB nor Bitcoin - it requires proper incentivisation model for cooperating agents. Sharing profits is more about human nature, nature of business and legal structuring than about underlying technology. 

So, to sum up: the idea can be viable, but probably in a different form  and after many details are thought through :)

**Reply by Goosie** [reply]
Thanks for the great feedback, enough to think it over :-)
